### ISSUE TITLE
Top level bounding box contains numpy value breaking export_html

### ISSUE THREAD

**Issue by bernhard-42** [bernhard-42/jupyter-cadquery#84]
A quick fix for https://github.com/bernhard-42/cad-viewer-widget/issues/3 is

```python
from jupyter_cadquery.ocp_utils import BoundingBox

def to_dict(self):
    return {
        "xmin": float(self.xmin),
        "xmax": float(self.xmax),
        "ymin": float(self.ymin),
        "ymax": float(self.ymax),
        "zmin": float(self.zmin),
        "zmax": float(self.zmax),
    }
BoundingBox.to_dict = to_dict
```

**Comment by bernhard-42** [comment]
CC @shimwell

**Comment by bernhard-42** [comment]
@shimwell I've released version 3.2 which fixes your issue with exporting html

**Comment by shimwell** [comment]
super super, perfect timing. Thanks very much for this
